tutorials/pretrain.md

```diff
@@ -36,6 +36,90 @@ litgpt pretrain \
 ```
 
 
+&nbsp;
+## Pretrain on custom data
+
+The simplest way to get started with pretraining on a small custom dataset is by using the `TextFiles` data module, which lets you pretrain a dataset from a folder containing plain text files.
+
+&nbsp;
+
+> [!NOTE]
+> This approach assumes that you have already cleaned the text files, for example, removing any unwanted characters and inserting beginning-of-sequence and end-of-sequence tokens if applicable.
```

Comment:
I'm confused by

> and inserting beginning-of-sequence and end-of-sequence tokens if applicable.

If we ask this to the user, then you need to make sure that `bos=False, eos=False` is set in `encode` or else it might add them automatically depending on the tokenizer config

Reply:
My bad, I assumed that this was the default but yeah it depends on the config. My bad.